java/vector/src/main/java/org/apache/arrow/vector/BitVectorHelper.java

```diff
@@ -140,22 +140,23 @@ public static int getNullCount(final ArrowBuf validityBuffer, final int valueCou
     }
     int count = 0;
     final int sizeInBytes = getValidityBufferSize(valueCount);
+    final int remainder = valueCount % 8;
```

Comment:
would you mind adding a comment similar to what was there before like `// If value count is not a multiple of 8, then calculate number of used bits in the last byte`